static/script/devices/mediaplayer/live/restartablefix.js

```diff
@@ -0,0 +1,68 @@
+require.def(
```

Comment:
Also needs license comment block at top of file, and ideally explanation of what its fixing and why.